### PR TITLE
Add document for tiup-cluster

### PR DESCRIPTION
**Pull request by lucklove**
document for https://github.com/pingcap/tiup/pull/615

Signed-off-by: lucklove <gnu.crazier@gmail.com>

### What is changed, added or deleted? (Required)
Add document for tiup-cluster

We introduce a new feature which can use native ssh client now.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tiup/pull/615

### CODE REVIEW

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -582,3 +590,29 @@ tiup cluster check <cluster-name> --cluster
 在运行检测时，若指定了 `--apply` 参数，程序将尝试对其中未通过的项目自动修复。自动修复仅限于部分可通过修改配置或系统参数调整的项目，其它未修复的项目需要根据实际情况手工处理。
 
 环境检查不是部署集群的必需流程。对于生产环境建议在部署前执行环境检查并通过所有检测项。如果未通过全部检查项，也可能正常部署和运行集群，但可能无法获得最佳性能表现。
+
+## 使用中控机系统自带的 SSH 客户端连接集群
+
+在以上所有操作中，涉及到对集群机器的操作都是通过 TiUP 内置的 SSH 客户端连接集群，但是在某些场景下，我们需要使用系统自带的 SSH 客户端来对集群执行操作，这时候我们可以使用 `--native-ssh` 参数，例如：
```
Comment (lonng) [inline tiup/tiup-cluster.md:596]: 可以加一个备注，说明某些场景是什么，举一些例子。

Comment (lonng) [reply tiup/tiup-cluster.md:596]: 另外文档不要太口语化，尽量避免“我们”。例如：
```
在以上所有操作中，涉及到对集群机器的操作都是通过 TiUP 内置的 SSH 客户端连接集群，但是在某些场景下（例如包含自定义 SSH 认证插件），需要使用系统自带的 SSH 客户端来对集群执行操作，此时可以通过命令行参数 `--native-ssh` 启用系统自带命令行。
```

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -582,3 +590,29 @@ tiup cluster check <cluster-name> --cluster
...
+
+- 部署集群: `tiup cluster deploy <cluster-name> <version> <topo> --native-ssh`
+- 启动集群: `tiup cluster start <cluster-name> --native-ssh`
+- 升级集群: `tiup cluster upgrade ... --native-ssh`
+
+所有涉及集群操作的步骤都可以加上 `--native-ssh` 来使用系统自带的客户端。
+
+如果觉得这样每个命令都加 `--native-ssh` 太冗余，我们可以使用环境变量来指定:
```
Comment (lonng) [inline tiup/tiup-cluster.md:604]:
```suggestion
也可以使用环境变量 `TIUP_NATIVE_SSH` 来指定是否使用本地 SSH 客户端，避免每个命令都需要添加 `--native-ssh` 参数：
```

File: tiup/tiup-cluster.md [hunk 3]
```diff
@@ -582,3 +590,29 @@ tiup cluster check <cluster-name> --cluster
...
+
+- 部署集群: `tiup cluster deploy <cluster-name> <version> <topo> --native-ssh`
+- 启动集群: `tiup cluster start <cluster-name> --native-ssh`
+- 升级集群: `tiup cluster upgrade ... --native-ssh`
+
+所有涉及集群操作的步骤都可以加上 `--native-ssh` 来使用系统自带的客户端。
+
+也可以使用环境变量 `TIUP_NATIVE_SSH` 来指定是否使用本地 SSH 客户端，避免每个命令都需要添加 `--native-ssh` 参数：
+
+```sh
```
Comment (july2993) [inline tiup/tiup-cluster.md:606]: 写一个就好了

Comment (lucklove) [reply tiup/tiup-cluster.md:606]: 都写上吧。。。QA 测试也好参考一下

File: tiup/tiup-cluster.md [hunk 4]
```diff
@@ -582,3 +590,34 @@ tiup cluster check <cluster-name> --cluster
...
+export TIUP_NATIVE_SSh=1
+# 或者
+export TIUP_NATIVE_SSH=enable
+```
+
+若环境变量和 `--native-ssh` 同时指定，则以 `--native-ssh` 为准。
+
+> **注意：**
+>
+> 在 deploy 步骤中，若需要使用密码的方式连接 (-p)，或者密钥文件设置了 passphrase，则需要保证中控机上安装了 sshpass，否则连接时会报错。
```
Comment (TomShawn) [inline tiup/tiup-cluster.md:623]:
```suggestion
> 在部署集群的步骤中，若需要使用密码的方式连接 (-p)，或者密钥文件设置了 passphrase，则需要保证中控机上安装了 sshpass，否则连接时会报错。
```

**Comment by lucklove** [comment]
how to merge?

**Pull request merged**